test/e2e/common/05_tektonhubdeployment_test.go

```diff
@@ -110,6 +110,19 @@ func (s *TektonHubTestSuite) SetupTest() {
 	s.logger.Debug("removing the tekton hub cr if any")
 	s.undeploy("")
 	s.undeployExternalDatabase()
+	// wait for target namespace to be deleted
+	interval := 3 * time.Second
+	timeout := 2 * time.Minute
+	er := resources.WaitForNamespaceDeletion(s.clients.KubeClient, s.resourceNames.TargetNamespace, interval, timeout)
+	if er != nil {
+		s.logger.Debugw("target namespace already cleanedup",
+			"namespace", s.resourceNames.TargetNamespace,
+		)
+	} else {
+		s.logger.Debugw("target namespace removed",
+			"namespace", s.resourceNames.TargetNamespace,
+		)
+	}
```

Comment:
@Basavaraju-G we have to add delete and wait under `undeploy` method,
```
err = resources.DeleteNamespaceAndWait(s.clients.KubeClient, s.resourceNames.TargetNamespace, pollInterval, timeout)
require.NoError(t, err, "Namespace deletion failed, namespace: %s",  s.resourceNames.TargetNamespace)
```

Reply:
@jkandasa thanks, i will check.